test/test-utils.js

```diff
@@ -22,8 +23,35 @@ const addCustomMintableTokenToExchange = async function(exchange, symbol, decima
     token: token,
   }
 }
+const deploySafe = async function(gnosisSafeMasterCopy, proxyFactory, owners, threshold) {
```

Comment:
Should we use here your new logic as well? 
using it in all our tests would give us more confidence.

Although, I am fine with using the old logic, as other tests are already in place for the new proxy deployments...

Reply:
The new logic is already used everywhere, this function should be only used to create the master safe now.